pkg/services/cloudmigration/cloudmigrationimpl/cloudmigration_test.go

```diff
@@ -790,7 +791,15 @@ func TestGetPlugins(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	t.Cleanup(cancel)
 
-	user := &user.SignedInUser{OrgID: 1}
+	user := &user.SignedInUser{
+		OrgID: 1,
+		Permissions: map[int64]map[string][]string{
+			1: {
+				pluginaccesscontrol.ActionInstall: {pluginaccesscontrol.ScopeProvider.GetResourceAllScope()},
```

Comment:
Using the real access control implementation rather than a fake, so I had to update the permission here